project-base/tests/ShopBundle/Database/Model/Category/CategoryDomainTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Tests\ShopBundle\Database\Model\Category;
+
+use Shopsys\FrameworkBundle\Model\Category\Category;
+use Shopsys\FrameworkBundle\Model\Category\CategoryDataFactory;
+use Shopsys\FrameworkBundle\Model\Category\CategoryFactory;
+use Tests\ShopBundle\Test\DatabaseTestCase;
+
+class CategoryDomainTest extends DatabaseTestCase
+{
+    const FIRST_DOMAIN_ID = 1;
+    const SECOND_DOMAIN_ID = 2;
+    const DEMONSTRATIVE_SEO_TITLE = 'Demonstrative seo title';
+    const DEMONSTRATIVE_SEO_META_DESCRIPTION = 'Demonstrative seo description';
+    const DEMONSTRATIVE_SEO_H1 = 'Demonstrative seo H1';
```

Comment:
I would personally avoid using constants for these testing strings, but it's OK to do so.